Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM golang:1.14-alpine AS build
+WORKDIR /go/src/saschagrunert/seccomp-operator
+RUN apk --no-cache add build-base git bzr mercurial gcc
+
+# Speed up build by leveraging docker layer caching
+COPY go.mod go.sum ./
+RUN go mod download
+
+ADD . /go/src/saschagrunert/seccomp-operator
+RUN go build -o /seccomp-operator
+
+
+
+FROM alpine
```

Comment:
Do we want to use `scratch` and link statically?

Reply:
I had some issues running from scratch, so was thinking on optimising (and also locking it down) once we have some the MVP functionality done and some automated E2E. What do you think?

Reply:
Sounds good, let's leave it like it is for now.